dmutils/user.py

```diff
@@ -59,10 +61,13 @@ def from_json(user_json):
         supplier_id = None
         supplier_name = None
         supplier_organisation_size = None
+        userResearchOptedIn = user['userResearchOptedIn'] if user['userResearchOptedIn'] else None
```

Comment:
I think this could be simplified further to `user_research_opted_in = user.get('userResearchOptedIn')`? (Note the snake case for the Python var...)

Reply:
thanks (didn't know it was a thing in python) :) - updated

Reply:
No worries 😸 In fact we could probably get rid of the var altogether and do the `user.get`  when creating the User object on line 81.

Reply:
@katstevens updated